procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -0,0 +1,631 @@
+{
```

Comment:
<p>I think this can be removed no?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
probably. I wish there was a `black` equivalent to catch this stuff.